disnake/onboarding.py

```diff
@@ -185,12 +253,69 @@ def __repr__(self) -> str:
             f"description={self.description!r} emoji={self.emoji!r}>"
         )
 
+    @classmethod
+    def _from_dict(
+        cls, *, data: OnboardingPromptOptionPayload, state: ConnectionState, guild: Guild
+    ) -> Self:
+        emoji = state._get_emoji_from_fields(
+            name=data.get("emoji_name"),
+            id=_get_as_snowflake(data, "emoji_id"),
+            animated=data.get("emoji_animated", None),
+        )
```

Comment:
```suggestion
        if emoji_data := data.get("emoji"):
            emoji = guild._state._get_emoji_from_data(emoji_data)
        else:
            emoji = None
```

It's not entirely clear from the docs, but we receive an `emoji` field here, the separate `emoji_*` fields are only used for sending
~~(except audit logs, where you get both for some reason.)~~

Reply:
78a5e2bf05730a7e8716a80af1a71b9edddef491